From 55f93c5364201311c3a468cce2c7297e3a64207f Mon Sep 17 00:00:00 2001
From: Kevron Rees <kevron.m.rees@intel.com>
Date: Mon, 11 Mar 2019 08:59:07 -0700
Subject: [PATCH] add -Wl and --no-undefined

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0162ba7..f128db1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -74,7 +74,7 @@ endif (WIN32)
 
 # set that name of the main output file as a target name
 if (NOT DEFINED COMMON_CLANG_LIBRARY_NAME)
-    set(COMMON_CLANG_LIBRARY_NAME common_clang)
+    set(COMMON_CLANG_LIBRARY_NAME opencl_clang)
 endif()
 set(TARGET_NAME ${COMMON_CLANG_LIBRARY_NAME}${BUILD_PLATFORM} )
 
@@ -305,6 +305,9 @@ if (WIN32)
             "RC_COPYRIGHT=\"Copyright ${RC_CHAR_C} 2018 Intel Corporation. All rights reserved.\"")
 endif(WIN32)
 
+set_property(TARGET ${TARGET_NAME} APPEND_STRING PROPERTY
+                LINK_FLAGS " -Wl,--no-undefined")
+
 install(FILES common_clang.h
         DESTINATION include/cclang
         COMPONENT ${TARGET_NAME})
-- 
2.17.1

