From 0de0c38e82493460f06ca113314039a495864b08 Mon Sep 17 00:00:00 2001
From: Kevron Rees <kevron.m.rees@intel.com>
Date: Tue, 12 Mar 2019 13:31:41 -0700
Subject: [PATCH] set project name to opencl_clang

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e1b520b..f128db1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -74,7 +74,7 @@ endif (WIN32)
 
 # set that name of the main output file as a target name
 if (NOT DEFINED COMMON_CLANG_LIBRARY_NAME)
-    set(COMMON_CLANG_LIBRARY_NAME common_clang)
+    set(COMMON_CLANG_LIBRARY_NAME opencl_clang)
 endif()
 set(TARGET_NAME ${COMMON_CLANG_LIBRARY_NAME}${BUILD_PLATFORM} )
 
-- 
2.17.1

