From 0ddd0f9df5dcc107de9390dcaeab461ab024424f Mon Sep 17 00:00:00 2001
From: Kevron Rees <kevron.m.rees@intel.com>
Date: Mon, 11 Mar 2019 08:59:07 -0700
Subject: [PATCH] add -Wl and --no-undefined

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0162ba7..e1b520b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -305,6 +305,9 @@ if (WIN32)
             "RC_COPYRIGHT=\"Copyright ${RC_CHAR_C} 2018 Intel Corporation. All rights reserved.\"")
 endif(WIN32)
 
+set_property(TARGET ${TARGET_NAME} APPEND_STRING PROPERTY
+                LINK_FLAGS " -Wl,--no-undefined")
+
 install(FILES common_clang.h
         DESTINATION include/cclang
         COMPONENT ${TARGET_NAME})
-- 
2.17.1

